### DISCUSSION TITLE
Funding

### DISCUSSION THREAD

**Discussion by softexpert** [davmac314/dinit#402] [General]
Hello,

I just saw that [FLOSS/fund](https://floss.fund/) offers the possibility to obtain funding for opensource projects.
Would you be interested in submitting an application - in the worst case the project is listed in the directory and perhaps finds additional supporters ?
Just a thought ...

Cheers !

**Comment by davmac314** [comment]
Hey, thanks for the suggestion. Doesn't look like we really meet their criteria for funding (at least not yet) but maybe I'll try to find time to put in application anyway at some point.
